### PR TITLE
Add code to translate Policy compliance results

### LINKED ISSUE #49: Map 'Policy Compliance' entries for nessus_mapper
The current version of nessus_mapper maps all plugin families except 'Policy Compliance'

Add capacity to map `Policy Compliance` plugins as well
Task list
- [ ] Identify STIG based plugins
- [ ] Parse CAT severity for STIG based plugins
- [ ] Parse NIST from compliance-reference for STIG based plugins
- [ ] Parse Vuln-ID from compliance-reference for STIG based plugins
- [ ] Use CAT severity to override plugin severity for STIG based plugins
- [ ] Parse Title
- [ ] Parse Checktext
- [ ] Parse Title
- [ ] Parse Results

### PR DESCRIPTION
**Pull request by rx294**
Fixes #49 

Nessus results field set are different for 'Policy Compliance' plug-in family vs other plug-in families Current version covers STIG based 'Policy Compliance' results

TODO Cover cases for 'Policy Compliance' results based on CIS #52

### CODE REVIEW

File: lib/heimdall_tools/nessus_mapper.rb [hunk 1]
```diff
@@ -96,14 +117,15 @@ def nist_tag(pluginfamily, pluginid)
     end
 
     def impact(severity)
+      # Map CAT levels and Plugin severity to HDF impact levels
       case severity
       when "0"
         IMPACT_MAPPING[:Info]
-      when "1"
+      when "1","III"
```
Comment (aaronlippold) [inline lib/heimdall_tools/nessus_mapper.rb:124]: This is incorrect, a CAT I or I is high ...
A CAT III or 3 is low

File: lib/heimdall_tools/nessus_mapper.rb [hunk 2]
```diff
@@ -96,14 +117,15 @@ def nist_tag(pluginfamily, pluginid)
...
         IMPACT_MAPPING[:Low]
-      when "2"
+      when "2","II"
         IMPACT_MAPPING[:Medium]
-      when "3"
+      when "3","I"
         IMPACT_MAPPING[:High]
       when "4"
         IMPACT_MAPPING[:Critical]
```
Comment (aaronlippold) [inline lib/heimdall_tools/nessus_mapper.rb:131]: CAT 4 or IIII is 'none' or 'info'

**Comment by aaronlippold** [comment]
No tests, no docs ... -> no MERGE

File: lib/heimdall_tools/nessus_mapper.rb (hunk 1 above)
Comment (rx294) [reply lib/heimdall_tools/nessus_mapper.rb:124]: Yep thats the CAT logic coded here
"III" -> low
"II" -> medium
"I" -> high

File: lib/heimdall_tools/nessus_mapper.rb (hunk 2 above)
Comment (rx294) [reply lib/heimdall_tools/nessus_mapper.rb:131]: Nessus report has no concept for CAT 4;
`info` level is placed based on the `plugin severity` 0

File: lib/heimdall_tools/nessus_mapper.rb (hunk 1 above)
Comment (rx294) [reply lib/heimdall_tools/nessus_mapper.rb:124]: The method is designed to make it work with both plugin severity and Nessus CAT reference.

Comment (aaronlippold) [reply lib/heimdall_tools/nessus_mapper.rb:124]: Ok

**Pull request merged**